### PR TITLE
Components/menu section

### PR DESCRIPTION
**Pull request by OsamaAbdellateef**

### CODE REVIEW

**Review by sebald — DISMISSED** [review]

File: themes/theme-b2b/src/components/NumberField.style.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const NumberField: Theme['components']['NumberField'] = {
         borderRight: '1px solid',
         borderColor: 'gray40',
       },
-      '&:nth-of-type(2)': {
```
[the remark this answers is not in the document]
Comment (OsamaAbdellateef) [reply themes/theme-b2b/src/components/NumberField.style.ts]: Yes sure, i fixed it on a new branch called "fix/NumberField" and i made a PR for this .

File: packages/components/src/Menu/Menu.tsx [hunk 2]
```diff
@@ -60,18 +61,31 @@ export const Menu = ({ variant, size, ...props }: MenuProps) => {
...
-        />
-      ))}
+      {[...state.collection].map(item => {
+        if (item.type === 'section') {
+          return (
+            <MenuSection
+              key={item.key}
+              item={item}
+              state={state}
+              onAction={props.onAction}
```
[the remark this answers is not in the document]
Comment (OsamaAbdellateef) [reply packages/components/src/Menu/Menu.tsx:71]: I added a css Property for MenuSection Component just like MenuItem but the section has no style for now , but it is already added if we want to add any style in the future .

File: docs/src/demos/components/Menu/menu-section.demo.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Menu } from '@marigold/components';
+
+export const MenuWithSections = () => {
```
Comment (sarahgm) [inline docs/src/demos/components/Menu/menu-section.demo.tsx:4]: I would like if there is also a Menu.Trigger around this :)

Comment (OsamaAbdellateef) [reply docs/src/demos/components/Menu/menu-section.demo.tsx:4]: **`Menu.Trigger`** has been added to the documentation demo .

**Review by sarahgm — APPROVED** [review]

**Pull request merged**